packages/core-flows/src/definition/cart/steps/retrieve-cart.ts

```diff
@@ -0,0 +1,43 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { ICartModuleService } from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+interface StepInput {
+  cartId: string
```

Comment:
todo: Add config to increase reusability of the step
```suggestion
  cartId: string
  config: FindConfig<CartDTO>
```

Reply:
nice!